mne/preprocessing/bads.py

```diff
@@ -20,6 +20,9 @@ def find_outliers(X, threshold=3.0, max_iter=2):
         The value above which a feature is classified as outlier.
     max_iter : int
         The maximum number of iterations.
+    tail : {"two", "positive", "negative"}
```

Comment:
to match functions like permutation_cluster_test I would use tail = 0, 1 or -1
it's for consistency although it's not the most beautiful API i agree